l10n/en-us.ftl

```diff
@@ -0,0 +1,102 @@
+baseline_supported_in = Supported in { $browsers }
+baseline_unsupported_in = Not widely supported in { $browsers }
+baseline_supported_and_unsupported_in = Supported in { $supported }, but not widely supported in { $unsupported }
+
+homepage-hero_title = Resources for <u data-l10n-name="developers">Developers</u>,<br> by Developers
+homepage-hero_description = Documenting <a data-l10n-name="css">CSS</a>, <a data-l10n-name="html">HTML</a>, and <a data-l10n-name="js">JavaScript</a>, since 2005.
+
+reference_toc_header = In this article
+
+footer_mofo = Visit <a data-l10n-name="moco">Mozilla Corporation’s</a> not-for-profit parent, the <a data-l10n-name="mofo">Mozilla Foundation</a>.
+footer_copyright = Portions of this content are ©1998–2024 by individual mozilla.org contributors. Content available under <a data-l10n-name="cc">a Creative Commons license</a>.
+
+search_title = Search results for: <em>{ $query }</em>
+search_stats = Found { $results } matches in { $time } milliseconds.
+
+search-modal_site-search = Site search for <em>{ $query }</em>
+
+blog_time_to_read = { $minutes ->
+    [one]   { $minutes } minute read
+   *[other] { $minutes } minutes read
+}
+blog_post_not_found = Blog post not found.
+
+blog_previous = Previous Post
+blog_next = Next Post
+
+Report = Report
+
+obs_title = HTTP Observatory
+obs_landing_intro = Launched in 2016, the HTTP Observatory enhances web security by analyzing compliance with best security practices. It has provided insights to over 6.9 million websites through 47 million scans.
+obs_assessment = Developed by Mozilla, the HTTP Observatory performs an in-depth assessment of a site’s HTTP headers and other key security configurations.
+obs_scanning = Its automated scanning process provides developers and website administrators with detailed, actionable feedback, focusing on identifying and addressing potential security vulnerabilities.
+obs_security = The tool is instrumental in helping developers and website administrators strengthen their sites against common security threats in a constantly advancing digital environment.
+obs_mdn = The HTTP Observatory provides effective security insights, guided by Mozilla's expertise and commitment to a safer and more secure internet and based on well-established trends and guidelines.
+
+
+compat_loading = Loading…
+
+compat_browser_version_date = { $browser } { $version } – Released { $date }
+
+compat_link_report_issue = Report problems with this compatibility data
+compat_link_report_issue_title = Report an issue with this compatibility data
+compat_link_report_missing_title = Report missing compatibility data
+compat_link_report_missing = Report this issue
+compat_link_source = View data on GitHub
+compat_link_source_title = File: { $filename }
+
+compat_deprecated = Deprecated
+compat_experimental = Experimental
+compat_nonstandard = Non-standard
+compat_no = No
```

Comment:
This likely needs a comment to say what "No" is refuting here (e.g. is this refusing some request, or is it short for `No compatibility` or something else?)